ddl/db_test.go

```diff
@@ -3203,6 +3203,130 @@ LOOP:
 	s.tk.MustExec("drop table partition_drop_idx;")
 }
 
+func (s *testDBSuite) TestPartitionCancelAddIndex(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.mustExec(c, "use test_db")
+	s.mustExec(c, "set @@session.tidb_enable_table_partition=1;")
```

Comment:
should we set @@session.tidb_enable_table_partition=0; at the end of this function.

Reply:
It's should be OK to `set @@session.tidb_enable_table_partition=1; ` for any test code. @XuHuaiyu 
We'll enable table partition by default later.